lib/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Langchain
-  VERSION = "0.3.12"
+  VERSION = "0.3.13"
```

Comment:
No need to update the version, I update it whenever I'm actually pushing out a new version.